BUILD.md

```diff
@@ -0,0 +1,88 @@
+# Building DLAF
+
+## Build using Spack
+
+Example installation:
+
+`spack install dla-future ^[virtuals=blas,lapack] intel-oneapi-mkl`
+
+or you can also build C API compatible with ScaLAPACK with:
+
+`spack install dla-future +scalapack ^[virtuals=blas,lapack,scalapack] intel-oneapi-mkl`
+
+You can go even further with a more detailed spec like this one, which builds dla-future in debug mode, using the clang compiler, specifying that the pika on which it depends has to be built
```

Comment:
```suggestion
You can go even further with a more detailed spec like this one, which builds `dla-future` in debug mode, using the clang compiler, specifying that the `pika` on which it depends has to be built
```

Reply:
This conflicts with a change suggested by @msimberg...

Reply:
This change is also fine by me.

Reply:
Ok switched to Rocco's suggestion 0dc7403196bbb2a304b340f5bf850fdf3ebf821b